crates/runtime/benches/queries/README.md

```diff
@@ -0,0 +1,23 @@
+# Spice.ai OSS Benchmarks
```

Comment:
we should make sure these end up in docs when we're finished. Let's make an issue in the docs repo if there isn't?